contracts/ResilientOracle.sol

```diff
@@ -169,7 +169,7 @@ contract ResilientOracle is OwnableUpgradeable, PausableUpgradeable {
      * @param vToken vToken address
      * @return price USD price in 18 decimals
      */
-    function getUnderlyingPrice(address vToken) external view returns (uint256) {
+    function getUnderlyingPrice(address vToken) external returns (uint256) {
```

Comment:
I am new to smart contracts and their concepts. But from the diff I don't see a change in the function so shouldn't the `view` stay here?

Reply:
If we put view there then compilation will fail because view functions aren't supposed to modify the state.